test/utils/dump.cpp

```diff
@@ -28,7 +28,7 @@ void dump(const evmone::code_analysis& analysis)
 
         if (c == OPX_BEGINBLOCK)
         {
-            block = &instr.arg.block;
+            block = &instr.block;
```

Comment:
I didn't get how this works without changing the way you iterate over `analysis.instrs`
(`instr` points here to the union containing `fn`, not `block_info`, right?)

Reply:
Also I'm not sure what `c` is here actually. Least significant byte of function pointer?